app/jobs/pair_request/auto_reject_job.rb

```diff
@@ -0,0 +1,12 @@
+class PairRequest
+  class AutoRejectJob < ApplicationJob
+    sidekiq_options queue: 'low'
+
+    # rubocop:disable Rails/SkipsModelValidations
+    def perform
+      PairRequest.pending.where('pair_requests.when < ?',
+        Time.current).update_all(status: :rejected)
```

Comment:
One thing I was thinking about here: We have an `:expired` status for pair requests. To me it might seem more semantically correct to use it for this situation compared to `:rejected`, which feels like a status better kept to when one of the request's owners actively turns the request down.

Reply:
ahh good call, yes i'll change.